solc/CommandLineInterface.cpp

```diff
@@ -778,37 +778,38 @@ void CommandLineInterface::actOnInput()
 
 bool CommandLineInterface::link()
 {
+	map<string, h160> librariesReplacements;
+	for (auto const& library: m_libraries)
+	{
+		string const& name = library.first;
+		string replacement = "__";
+		for (size_t i = 0; i < 36; ++i)
```

Comment:
Can you add a comment explaining what it does. It is hard do guess, especially with this magic 36 and 40.